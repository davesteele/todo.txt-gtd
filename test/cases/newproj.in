# _None

new taskproj @computer +NewProj
