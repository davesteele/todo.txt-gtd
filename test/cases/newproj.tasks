
@computer
new taskproj @computer +NewProj
