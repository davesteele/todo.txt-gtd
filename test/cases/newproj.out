# NewProj
#

new taskproj @computer +NewProj

# _None

